### DISCUSSION TITLE
Cursor hight & interline in editor

### DISCUSSION THREAD

**Discussion by atakito** [zed-industries/zed#12825] [Feature Requests]
Hello. 
Is there any place on settings, to adjust interline space and the cursor hight ? 
thanks.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
